src/--tests--/searchbar.test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import SearchBar from '../SearchBar'
+// import { shallow, mount, render } from 'enzyme'
+import { shallow } from 'enzyme'
+
+/* global it describe expect beforeEach jest */
+
+describe('SearchBar', () => {
+  let wrapper
+  beforeEach(() => {
+    const myFakeFunc = jest.fn()
+    wrapper = shallow(<SearchBar
+      handleSearchString={myFakeFunc}
+      handleFilterStockedToggle={myFakeFunc}
+    />)
```

Comment:
Looking good! This is still probably in progress, but you're going to want to use different faked (or "mocked") functions for each of the two callbacks.